osu.Game.Rulesets.Osu/Mods/OsuModClassic.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Rulesets.Osu.Objects.Drawables;
+using osu.Game.Rulesets.Osu.UI;
+using osu.Game.Rulesets.UI;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    public class OsuModClassic : Mod, IApplicableToHitObject, IApplicableToDrawableHitObjects, IApplicableToDrawableRuleset<OsuHitObject>
+    {
+        public override string Name => "Classic";
+
+        public override string Acronym => "CL";
+
+        public override double ScoreMultiplier => 1;
+
+        public override IconUsage? Icon => FontAwesome.Solid.History;
+
+        public override string Description => "Feeling nostalgic?";
+
+        public override bool Ranked => false;
+
+        public override ModType Type => ModType.Conversion;
+
+        [SettingSource("Disable slider head judgement", "Scores sliders proportionally to the number of ticks hit.")]
+        public Bindable<bool> DisableSliderHeadJudgement { get; } = new BindableBool(true);
+
+        [SettingSource("Disable slider head tracking", "Pins slider heads at their starting position, regardless of time.")]
```

Comment:
```suggestion
        [SettingSource("No slider head movement", "Keep slider heads stationary at their starting position, instead of following the ball.")]
```